From 865920c3e82186d981c33340e47fbfd7e4c014e0 Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@nxp.com>
Date: Mon, 9 Jul 2018 19:47:58 +0800
Subject: [PATCH 4280/5242] MLK-18816-4: arm64: dts: imx8mq-evk: enable touch

commit  d4570f31125a9bbf2bb986b010bf28b0ae222e39 from
https://source.codeaurora.org/external/imx/linux-imx.git

Enable synaptics touch on imx8mq-evk board when connect the
MIPI DSI panel.

Signed-off-by: Haibo Chen <haibo.chen@nxp.com>
(cherry picked from commit 17e25d8fd314de9628a599466508864678e76d6a)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dts  |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dts
index 78fa6d4..6aa6e9e 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dts
@@ -132,7 +132,6 @@
 		pinctrl-0 = <&pinctrl_i2c1_synaptics_dsx_io>;
 		interrupt-parent = <&gpio5>;
 		interrupts = <7 8>;
-	/*	rest-gpios = <&gpio5 6 GPIO_ACTIVE_HIGH>;  */
-		status = "disabled";
+		status = "okay";
 	};
 };
-- 
1.7.9.5

